packages/host/package.json

```diff
@@ -171,6 +171,7 @@
     ]
   },
   "dependencies": {
-    "ember-async-data": "^1.0.1"
+    "ember-async-data": "^1.0.1",
+    "ember-composable-helpers": "^5.0.0"
```

Comment:
Is this only for {{map}}? Maybe we can just use a getter rather than introducing a new package

Reply:
Yes, it is for the map.

Reply:
Agreed.

Reply:
updated [c109af5](https://github.com/cardstack/boxel/pull/554/commits/c109af5366dde34bdd67ae9e82d9f1e7286abd8f)